includes/polling/os/zxdsl.inc.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+use LibreNMS\RRD\RrdDefinition;
```

Comment:
You don't even need this statement here. There is no RRD work done in this file.

Reply:
Forgotten to remove this line, fixed